tests/components/axis/test_device.py

```diff
@@ -157,14 +172,14 @@ async def test_device_not_accessible(hass):
     """Failed setup schedules a retry of setup."""
     with patch.object(axis.device, "get_device", side_effect=axis.errors.CannotConnect):
         await setup_axis_integration(hass)
-    assert hass.data[axis.DOMAIN] == {}
+    assert hass.data[AXIS_DOMAIN] == {}
 
 
 async def test_device_unknown_error(hass):
     """Unknown errors are handled."""
     with patch.object(axis.device, "get_device", side_effect=Exception):
         await setup_axis_integration(hass)
-    assert hass.data[axis.DOMAIN] == {}
+    assert hass.data[AXIS_DOMAIN] == {}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)